site/src/pages/components/invokers.explainer.mdx

```diff
@@ -311,6 +311,8 @@ Invokers, the names and exact semantics may be subject to change:
 | `<*>`           | `'toggleFullscreen'`  | If the element is fullscreen, then exit, otherwise request to enter                  |
 | `<*>`           | `'requestFullscreen'` | Request the element to enter into 'fullscreen' mode                                  |
 | `<*>`           | `'exitFullscreen'`    | Request the element to exit 'fullscreen' mode                                        |
+| `<input type=number>` | `'stepUp'`      | Call `.stepUp()` on the invokee                                                      |
```

Comment:
stepUp and stepDown take an argument which is multiplied by the step attribute. Should invokers handle that somehow? Thinking maybe the button value attribute can be used as the argument?

We have precedence for the dialog returnValue?

Reply:
I think it'd be best to avoid getting more complicated on things like this. The number already has a step, so the most natural argument is `1` - step by 1.